cloud-sql/sqlserver/activerecord/config/database.yml

```diff
@@ -46,15 +46,10 @@ development:
 # Do not set this db to the same as development or production.
 
 test:
-  adapter: sqlserver
-  pool: 5
+  adapter: sqlite3
```

Comment:
Probably prefer to use the proxy when running locally?

Reply:
It's using the proxy in the development db, but not for tests. I originally had this set to use the proxy as well, but it turns out that breaks a bunch of things in the tests, since the CI isn't set up to run the proxy (we probably should fix that, since imo there really isn't a point to testing connection samples unless you're checking to see if they can actually connect)

Reply:
SGTM